subsys/fs/shell.c

```diff
@@ -99,7 +99,7 @@ static int cmd_cd(const struct shell *shell, size_t argc, char **argv)
 
 	if (entry.type != FS_DIR_ENTRY_DIR) {
 		shell_fprintf(shell, SHELL_ERROR,
-				"%s is not a directory\n", path);
+			      "%s is not a directory\n", path);
```

Comment:
you can simplify it even more by using macro `shell_error`

Instead of writing:

```
shell_fprintf(shell, SHELL_ERROR,
	      "%s is not a directory\n", path);
```

you can write:

`shell_error(shell, "%s is not a directory\n", path);`

Reply:
@jarz-nordic : I have change all logs to shell_error, additionally changed printk to shell_print also.
Pls check.